ui/src/app/views/workspace-edit/component.ts

```diff
@@ -48,17 +50,28 @@ export class WorkspaceEditComponent implements OnInit {
         population: false,
         reviewRequested: false
       }};
+    if (this.route.routeConfig.data.title === 'Create Workspace') {
+      this.adding = true;
+    } else {
+      this.oldWorkspaceNamespace = this.route.snapshot.url[1].path;
```

Comment:
Can we access path paramters via `ActivatedRouteSnapshot`, so we can reference them by name? https://angular.io/api/router/ActivatedRouteSnapshot